src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1542,7 +1631,64 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 			return NULL;
 #ifdef TARGET_WASM
 		return emit_simd_ins_for_sig (cfg, klass, OP_WASM_SIMD_BITMASK, -1, -1, fsig, args);
-#else
+#elif defined(TARGET_ARM64)
+		MonoInst* result_ins = NULL;
```

Comment:
If this is not working with LLVM, a condition needs to be added here. `if (!COMPILE_LLVM (cfg))`

Reply:
Thank you, I missed this. Fixed.